lang/cs/Org.Apache.REEF.IMRU.Examples/Run.cs

```diff
@@ -32,8 +32,121 @@ namespace Org.Apache.REEF.IMRU.Examples
     /// </summary>
     public class Run
     {
+        private static void RunMapperTest(IConfiguration tcpPortConfig, bool runOnYarn, int numNodes)
+        {
+            IInjector injector;
+
+            if (!runOnYarn)
+            {
+                injector =
+                    TangFactory.GetTang()
+                        .NewInjector(OnREEFIMRURunTimeConfiguration<int, int, int>.GetLocalIMRUConfiguration(numNodes), tcpPortConfig);
+            }
+            else
+            {
+                injector = TangFactory.GetTang()
+                    .NewInjector(OnREEFIMRURunTimeConfiguration<int, int, int>.GetYarnIMRUConfiguration(), tcpPortConfig);
+            }
+
+            var mapperCountExample = injector.GetInstance<MapperCount.MapperCount>();
+            mapperCountExample.Run(numNodes - 1);
+        }
+
+        private static void RunBroadcastReduceTest(IConfiguration tcpPortConfig, bool runOnYarn, int numNodes, string[] args)
+        {
+            int chunkSize = 2;
+            int dims = 10;
+            int iterations = 10;
+
+            if (args.Length > 0)
+            {
+                dims = Convert.ToInt32(args[0]);
+            }
+
+            if (args.Length > 1)
+            {
+                chunkSize = Convert.ToInt32(args[1]);
+            }
+
+            if (args.Length > 2)
+            {
+                iterations = Convert.ToInt32(args[2]);
+            }
+
+            IInjector injector;
+
+            if (!runOnYarn)
+            {
+                injector =
+                    TangFactory.GetTang()
+                        .NewInjector(OnREEFIMRURunTimeConfiguration<int[], int[], int[]>.GetLocalIMRUConfiguration(numNodes), tcpPortConfig);
+            }
+            else
+            {
+                injector = TangFactory.GetTang()
+                    .NewInjector(OnREEFIMRURunTimeConfiguration<int, int, int>.GetYarnIMRUConfiguration(), tcpPortConfig);
+            }
+            var broadcastReduceExample = injector.GetInstance<PipelinedBroadcastAndReduce>();
+            broadcastReduceExample.Run(numNodes - 1, chunkSize, iterations, dims);
+        }
+
         private static void Main(string[] args)
-        {      
+        {
+            Console.WriteLine("start running client: " + DateTime.Now);
```

Comment:
Please use logging instead. These calls might end up in tests on day, and being able to redirect the logs is helpful for those.